### PR TITLE
spark1.5

### PR DESCRIPTION
**Pull request by mgummelt**
Note that I removed the DCOS_CONFIG check.  The new CLI doesn't require DCOS_CONFIG

### CODE REVIEW

**Comment by mgummelt** [comment]
cc @tnachen

File: dcos_spark/discovery.py [hunk 1]
```diff
@@ -8,18 +8,12 @@
...
 from dcos import marathon, util
 
-def check_dcos_config():
-    if os.getenv("DCOS_CONFIG") is None:
-        print("Please set the DCOS_CONFIG environment variable to your DCOS config file path, e.g: ~/.dcos/dcos.toml")
-        sys.exit(1)
 
 def get_spark_webui():
-    check_dcos_config()
     base_url = util.get_config().get('core.dcos_url')
```
Comment (tnachen) [inline dcos_spark/discovery.py]: util.get_config() doesn't require DCOS_CONFIG anymore?

Comment (mgummelt) [inline dcos_spark/discovery.py]: No.  There's a default.

Comment (tnachen) [inline dcos_spark/discovery.py]: great

**Comment by tnachen** [comment]
LGTM

**Pull request merged**